src/views/RegionSeasonalRiskWatch/Filters/index.tsx

```diff
@@ -54,13 +56,48 @@ function Filters(props: Props) {
 
     const countryList = useCountry({ region: regionId });
 
+    useEffect(
+        () => {
+            // TODO: figure out better way to set default value
+            // NOTE: setting default value
+            onChange((prevValue) => ({
+                ...prevValue,
+                countries: countryList.map((country) => country.iso3),
+                months: Array.from(Array(11).keys()),
+                hazardTypes: mapToList(
+                    applicableHazardsByRiskMetric[prevValue.riskMetric],
+                    (isApplicable, hazardType) => (isApplicable
+                        ? hazardType as HazardType : undefined),
+                ).filter(isDefined),
+            }));
+        },
+        [countryList, onChange],
+    );
```

Comment:
We need a onDataLoad callback from useCountry.

Reply:
We'll need more than that since hazard Types are also dynamic. We need to figure out a way to reliably set the default values.